sfge/src/test/java/com/salesforce/rules/unusedmethod/UnusedMethodRuleTest_BaseClass.java

```diff
@@ -0,0 +1,131 @@
+package com.salesforce.rules.unusedmethod;
+
+import static org.hamcrest.Matchers.*;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import com.salesforce.TestUtil;
+import com.salesforce.graph.vertex.MethodVertex;
+import com.salesforce.rules.AbstractStaticRule;
+import com.salesforce.rules.UnusedMethodRule;
+import com.salesforce.rules.Violation;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+import java.util.function.Consumer;
+import org.apache.tinkerpop.gremlin.process.traversal.dsl.graph.GraphTraversalSource;
+import org.hamcrest.MatcherAssert;
+import org.junit.jupiter.api.BeforeEach;
+
+public class UnusedMethodRuleTest_BaseClass {
```

Comment:
This is really helpful. Based on core's naming standards, we named the equivalent of this for FLS rule as `BaseFlsTest`. Can we have a similar name for this? Something like BaseUnunusedMethodTest?

Reply:
The standard is that test classes end with the word "Test", and "Base" indicates that the class is an abstract layer, which encapsulates the common operations.